vanilla-framework/vanilla-dashboard-theme
# Modal

## Done

- Built an accessible modal to be used in the dashboard theme

## QA

- pull branch and do `./run`
- See new pattern here http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/modal/modal/
- Compare to design and spec https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/Modal%20window

## Details

- Fixes #10
